testing-suite/staking-v4/scenarios/PR_6114.py

```diff
@@ -0,0 +1,181 @@
+import threading
+import time
+
+from core.validatorKey import *
+from core.chain_simulator import *
+from chain_commander import *
+from staking import stake, unStake, unBondNodes
+from get_infos.get_validator_info import get_keys_state, get_keys_from_validator_statistics, \
+    get_keys_from_validator_auction
+import requests
+
+
+# SCENARIO 1
+# Have every epoch auction list with enough nodes (let's say 8 qualified, 2 unqualified)
+# Unstake 1 or more eligible/waiting nodes (no more than all nodes per shard or no more than all eligible)
```

Comment:
In this scenario you actually only unstaked one node, not more.
Also, could've helped to specify exactly what's the configuration for this test: like how many eligible/waiting your test is starting with

Reply:
Is copy-paste from your description from the PR. Just to have it here. My testcase is bellow and it's clear that we unstake 1 node. 

I added the configuration.